packages/SwingSet/src/vats/comms/controller.js

```diff
@@ -53,7 +55,9 @@ export function deliverToController(
     const remoteID = state.getRemoteIDForName(remoteName);
     assert(remoteID, X`unknown remote name ${remoteName}`);
     const remoteRefID = args[1];
-    const localRef = provideLocalForKernel(krefOf(args[2]));
+    const kernelRefID = krefOf(args[2]);
+    assert(kernelRefID, 'unexpected "obj0" arg for addEgress()');
```

Comment:
Echo the above comment.  I don't understand "unexpected".  We are expecting the arg.  I think you want a different word here but I'm not sure what it is.

Reply:
I'll say "bad" here too